### PR TITLE
Don't raise when TabbedComponent does not select a tab

### PR DESCRIPTION
**Pull request by manuelpuyol**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -116,7 +116,8 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.8)
```
Comment (srt32) [inline Gemfile.lock:119]: Is this change related?

**Review by srt32 — APPROVED** [review]
Could we add some explanation to the PR description?

**Review by joelhawksley — APPROVED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (manuelpuyol) [reply Gemfile.lock:119]: this was needed for CI to work (and to run test locally 😓 )
